addon/views/sl-split-grid-column-header.js

```diff
@@ -0,0 +1,105 @@
+import SlSplitGridCellView from './sl-split-grid-cell';
+
+/**
+ * @module views
+ * @class  sl-split-grid-column-header
+ */
+export default SlSplitGridCellView.extend({
+
+    // -------------------------------------------------------------------------
+    // Dependencies
+
+    // -------------------------------------------------------------------------
+    // Attributes
+
+    /**
+     * The HTML tag name of the view element
+     *
+     * @property {string} tagName
```

Comment:
All of the other files in this repository use Ember.String. Please change the occurrence for this property as well as for all other similar instances in this file to be consistent with the rest of the codebase. If it is desired to move away from the "Ember." syntax create an issue to be worked on separately.